examples/tutorial/04_Working_with_Tabular_Data.ipynb

```diff
@@ -109,9 +109,9 @@
    "source": [
     "## Declaring Dimensions on a Dataset\n",
     "\n",
-    "A HoloViews `Dataset` is similar to a HoloViews Element, without having any specific metadata that lets it visualize itself. A Dataset is useful for specifying a set of `Dimension`s that apply to the data, which will later be inherited by any visualizable elements that you create from the Dataset.\n",
+    "A HoloViews `Dataset` is similar to a HoloViews Element, but without having the full set of metadata that would let it visualize itself. A Dataset is useful for declaring the set of `Dimension`s along which your data varies, which will later be inherited by any more specific visualizable elements that you create from the Dataset.\n",
     "\n",
-    "A HoloViews Dimension is the same concept as a **dependent** or **independent** variable in mathematics. In HoloViews such variables are called value dimensions and key dimensions (respectively). In `macro_df`, the ``'country'`` and ``'year'`` are the independent variables, so when we create a Dataset we will declare those as key dimensions.  The remaining columns will automatically be inferred to be value dimensions:"
+    "A HoloViews `Dimension` is like a mathematical variable, i.e., a quantity that can have different values for different data points.  There are two types of dimensions: "key dimensions" (like mathematical independent variables), and "value dimensions" (like dependent variables). In `macro_df`, the ``country`` and ``year`` are the independent variables (values where measurements were taken), so when we create a Dataset we will declare those as key dimensions.  The remaining columns will automatically be inferred to be value dimensions (quantities measured for each combination of key dimension values):"
```

Comment:
The json is invalid here because of the double quotes at `"key dimensions"` and `"key dimensions"`. Should be an easy fix.

Reply:
```suggestion
    "A HoloViews `Dimension` is like a mathematical variable, i.e., a quantity that can have different values for different data points.  There are two types of dimensions: 'key dimensions' (like mathematical independent variables), and 'value dimensions' (like dependent variables). In `macro_df`, the ``country`` and ``year`` are the independent variables (values where measurements were taken), so when we create a Dataset we will declare those as key dimensions.  The remaining columns will automatically be inferred to be value dimensions (quantities measured for each combination of key dimension values):"
```